src/cogs/redditAPI.py

```diff
@@ -56,6 +67,131 @@ async def memes(self, ctx, arg : int, x = 1):
             em2 = discord.Embed(title = "Range Exceeded", description = "We are bound to provide you maximum 5 memes at a time", color = discord.Color.green())
             await ctx.send(embed = em2)
 
```

Comment:
try keeping variable names a bit more self-explanatory, for ex. using `embed_2` instead of `em2`.